main/reference/ertp-api/amount-math.md

```diff
@@ -183,21 +176,21 @@ AmountMath.isEqual(empty, quatloos10);
 - **brand**: **[Brand](./brand.md)** - Optional, defaults to **undefined**.
 - Returns: **Amount**
 
-Returns a new **Amount** that is the union of *leftAmount* and *rightAmount*. Both
-arguments must be of the same **Brand**.
+Returns a new **Amount** that is the combination of *leftAmount* and *rightAmount*.
+Both **Amount** arguments must have the same **Brand**.
 
 If the optional *brand* argument doesn't match the **Amount**s' **Brand**, an error is thrown.
 
 For fungible **Amounts** this means adding their **[AmountValues](./ertp-data-types.md#amountvalue)**. For non-fungible
 **Amounts**, it usually means including all of the elements from *leftAmount*
 and *rightAmount*.
 
-If either *leftAmount* or *rightAmount* is empty, this method returns the non-empty 
-**Amount** argument. If both are empty, this method returns an empty **Amount**.
+If one of *leftAmount* or *rightAmount* is empty, this method returns an **Amount** equivalent to the other.
+If both are empty, this method returns a new empty **Amount**.
```

Comment:
```suggestion
If both are empty, this method returns an empty **Amount**.
```

Likewise. The API spec should not say whether or not the empty amount is newly allocated or not.